README.md

```diff
@@ -0,0 +1,51 @@
+# dgraph-schema
+dgraph-schema is node a CLI allowing you to manage your Dgraph schemas with ease.
+You can:
+- Print your current Dgraph schema
+- Print the output between your new schema & the current one
+- Alter your Dgraph schema if there are no conflicts
+
+## Requirement
+Node v13.12.0 or later
+New schema in file named `schema.js` in the same folder as `cli.js`
+
+## Example of a schema
+```javascript
+// eslint-disable-next-line unicorn/filename-case
+export const SCHEMA = {
+  uuid: 'string @index(exact,fulltext) @upsert .',
+  hash: 'string .',
+  desc: 'int .',
+  name: '[string] .',
+}
+
+export const TYPES = {
+  user: ['uuid', 'hash', 'desc'],
+  pets: ['name'],
+}
+```
+
+
+## How to use it
+`node --experimental-specifier-resolution=node cli.js` to use the CLI.
```

Comment:
A cli is not ran through node but by itself

- [how to](https://levelup.gitconnected.com/how-to-build-a-cli-npm-package-3ba98d6f9d4e)
- [harmony flag](https://github.com/HydreIO/doubt/blob/master/src/cli.js)